code/__HELPERS/string_lists.dm

```diff
@@ -17,9 +17,9 @@ GLOBAL_LIST_EMPTY(string_lists)
 		return values //baseturf things
 	//	return values
 	if(length(values) > 10)
-		stack_trace("The baseturfs list of [baseturf_holder] at [baseturf_holder.x], [baseturf_holder.y], [baseturf_holder.z] is [length(values)], it should never be this long, investigate. I've set baseturfs to a flashing wall as a visual queue")
-		baseturf_holder.ChangeTurf(/turf/closed/indestructible/baseturfs_ded, list(/turf/closed/indestructible/baseturfs_ded), flags = CHANGETURF_FORCEOP)
-		return string_list(list(/turf/closed/indestructible/baseturfs_ded)) //I want this reported god damn it
+		stack_trace("The baseturfs list of [baseturf_holder] at [baseturf_holder.x], [baseturf_holder.y], [baseturf_holder.z] is [length(values)]. Values: [values.Join(", ")]")
+		//baseturf_holder.ChangeTurf(/turf/closed/indestructible/baseturfs_ded, list(/turf/closed/indestructible/baseturfs_ded), flags = CHANGETURF_FORCEOP)
+		//return string_list(list(/turf/closed/indestructible/baseturfs_ded)) //I want this reported god damn it
```

Comment:
Seems fine, revert the shit is fuck removal and I'll merge. We want things to obviously break when they do